opengrok-tools/setup.py

```diff
@@ -0,0 +1,58 @@
+import os
+import unittest
+from setuptools import setup
+
+
+def readme():
+    with open(os.path.join(os.path.dirname(os.path.realpath(__file__)), 'src', 'main', 'python', 'README.md'), 'r') as readme:
```

Comment:
this line looks a bit long, does the script pass `flake8` ?

Reply:
Maybe it’s excluded from the check because the flake tests are part of Travis and it succeeded. Let me check

Reply:
added